types/ember__object/events.d.ts

```diff
@@ -1,34 +1,32 @@
 import { ObserverMethod } from '@ember/object/-private/types';
 
+export {};
+
+type Callback = () => void;
```

Comment:
1) 

Some versions of typescript have problems when you use a non-exported type in an exported type/signature. You'll want this alias to live in a package that's unlikely to collide with anything the `@ember/object` package will contain. 

Look at the other `@types/ember__*` modules for examples. We usually put things like this in folders like `./-private/types.d.ts` or something.


2) 

If you want to create a type alias, the proper one that's equivalent to `Function` is 

```ts
(...args: any[]) => any
```

Reply:
I was using `Function` but dtslint was against me, so I created an alias type. And because I had an alias type it forced me to have this `export {}`. Very weird. The code I pushed now is what I had first, before dtslint.